### DISCUSSION TITLE
How Container Balancer uses Placement Policies and some suggestions for small clusters

### DISCUSSION THREAD

**Discussion by siddhantsangwan** [apache/ozone#6232] [General]
I wrote a small document that discusses the basics of how Container Balancer uses Ozone's Placement Policies to balance both Ratis and EC containers. Since many people try out Ozone on small clusters of less than 20 Datanodes, the document also has suggestions for running balancer in such clusters. Sharing the doc here in case it's useful for the community!

https://docs.google.com/document/d/1lyJncitr6pOgxneLEMfn3BzYwCr1TC-jzkfWOjMB7OU/edit?usp=sharing

**Comment by kerneltime** [comment]
@siddhantsangwan maybe a markdown text in the discussions itself? This will help search engines to pick up the information.

**Comment by siddhantsangwan** [comment]
@kerneltime good idea, I'll try that.
